dbt/parsely_dpl/dbt_project.yml

```diff
@@ -0,0 +1,55 @@
+
+# Name your package! Package names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'parsely'
+version: '0.0.1'
+
+# This setting configures which "profile" dbt uses for this project. Profiles contain
+# database connection information, and should be configured in the  ~/.dbt/profiles.yml file
+profile: 'parsely_blog' #'default'
```

Comment:
we'll want to remove the hardcoded items (i.e. parsely_blog, etc); i'll mark what I can see, but a grep might be a good idea.